apps/asap-server/test/handlers/research-outputs/fetch-by-id.test.ts

```diff
@@ -42,25 +41,9 @@ describe('GET /research-outputs/{id} - validations', () => {
   });
 
   test('returns 403 when Auth0 fails to verify token', async () => {
-    nock(`https://${authConfig.domain}`).get('/userinfo').reply(404);
-
-    const result = (await handler(
-      apiGatewayEvent({
-        httpMethod: 'get',
-        headers: {
-          Authorization: `Bearer token`,
-        },
-        pathParameters: {
-          id,
-        },
-      }),
-    )) as APIGatewayProxyResult;
-
-    expect(result.statusCode).toStrictEqual(403);
-  });
-
-  test('returns 403 when Auth0 is unavailable', async () => {
-    nock(`https://${authConfig.domain}`).get('/userinfo').reply(500);
+    jest
+      .requireMock('@asap-hub/auth')
```

Comment:
I'd recommend against using this API - it'll return the mock no matter if it is actually active or not - if it's not then it's extremely confusing to track down why this is not doing anything.
I'd import `decodeToken` normally instead and then cast `const mockDecodeToken = decodeToken as jest.MockedFunction<typeof decodeToken>`